Add Know Your Plugin screen (#13)
SyncSavings/integration-guide: docs/customing-your-plugin.md

---

> title: Customising your plugin

Reviewer: ```suggestion
title: Customising Your Plugin
```

nav_order: 7
layout: default
---

# Customising your Payroll Savings Plugin
{: .no_toc }

> In order to make your (or your customers) feel as much as home possible, this document will guide you through the areas of our **Payroll Savings Plugin** that can be customised.

Reviewer: This bit don't read right

> In order to make your (or your customers) feel as much as home possible

Maybe this is what we meant?

> In order to make your users (or your customers' users) feel as much at home as possible

> Use this guide in parallel with the provided themes data spreadsheet.

Reviewer: Is there a link to this? Or do people get sent it?

## Table of contents
{: .no_toc .text-delta }

1. TOC
{: toc }

## 🎨 Colors

**`Primary`** is applied extensively throughout the Plugin and is recommended to be a recognisable brand colour that would make the user most feel at home.

Below are number of examples of the application of the _primary_ colour in our own **Sync Turquoise**:

> ![Bullet Icons and Graphic base colour on the Landing Explainer Screen](/images/image.png)

Reviewer: Can we update the naming of these files please?

<img width="222" height="325" alt="Image" src="https://github.com/user-attachments/assets/65365234-a394-4189-96b3-d8e5afda06d7" />

Bullet Icons and Graphic base colour on the Landing Explainer Screen

![Border of a selected Card and the Progress Stepper (Breadcrumbs)](/images/image%201.png)

Border of a selected Card and the Progress Stepper (Breadcrumbs)

![Onboarding Complete screen’s NextSteps, Dashboard’s NextDeposit graphic and Transactions Item’s highlight text](/images/image%202.png)

Onboarding Complete screen’s NextSteps, Dashboard’s NextDeposit graphic and Transactions Item’s highlight text

## 🖱️ Buttons

> **`Button (optional)`** overrides the primary/action button background colour. We recommend leaving this field empty or matching buttons to the redirecting or embedding app for a truly seamless transition.

Reviewer: ```suggestion
**`Button (optional)`** overrides the primary/action button background colour. We recommend leaving this field empty or matching buttons to your app for a truly seamless transition.
```

Reviewer: And everywhere else as well

---
title: Customising your plugin

> nav_order: 5

Author: To enable quick merge of all page updates, I have pre-allocated all new page numbers for outstanding PRs
```suggestion
nav_order: 6
```
